FrameworkSystem/Client/ComponentInstaller.py

```diff
@@ -1598,7 +1599,8 @@ def setupSite(self, scriptCfg, cfg=None):
           cmdFound = True
       if not cmdFound:
         gLogger.notice('Starting runsvdir ...')
-        os.system("runsvdir %s 'log:  DIRAC runsv' &" % self.startDir)
```

Comment:
This fixes the issue in Jenkins about `Process leaked file descriptors`. The issue is that the forked process from `os.system` inherits the parents std(in/out/err) and then keeps these handles open indefinitely. This results in `docker exec server ./install-server.sh` hanging forever as it never receives EOF.